### DISCUSSION TITLE
Does deadbeef support Wayland, or are there plans to do so?

### DISCUSSION THREAD

**Discussion by miyl** [DeaDBeeF-Player/deadbeef#2599] [General]
See title.

**Comment by saivert** [comment]
Yes, Deadbeef runs on Wayland if you use the GTK3 UI plugin.
Global hotkeys wont work though.

**Reply by miyl** [reply]
Better late than never (my response): Thanks!

**Comment by pinballmachine** [comment]
Hotkeys do not work when configured from within Deadbeef, but when using KDE on Wayland, at least it is possible to use the general KDE shortcut setttings, add Deadbeef as an application, and set hotkeys for next, pause, play, previous and stop.

**Comment by w-jablonski** [comment]
Without xwayland installed, hotkeys do not work at all for me. Not just global hotkeys, but it seems any keyboard actions other than up/down to select track. This is despite Deadbeef running fully on Wayland. Is this a known issue or only happening for me? 1.9.5 Arch under sway. Thanks.

**Reply by Oleksiy-Yakovenko** [reply]
I wasn't aware of the issue, but it makes sense from the code perspective. Some work needs to be done to make it work properly. Please post a new issue.
